web-console/4.0-designs/user/user.md

```diff
@@ -0,0 +1,19 @@
+# User Menu
+
+## Copy Login Command
+![pic 0](img/copy-login-command-1.png)
+
+- Copy Login Command is the first option under the user menu in the masthead.
+
+![pic 1](img/copy-login-command.png)
+
+- Clicking this option opens a modal.
+- The oc login command is presented in conjunction with the user's unique token.
+- A warning is presented above the command to remind users not to share their token.
+
+![pic 3](img/copy-login-command-3.png)
```

Comment:
@beanh66 small nit, but can you change Logout to "Log out" ?  I know that Robb was putting that through this week (applies to all 3 images)